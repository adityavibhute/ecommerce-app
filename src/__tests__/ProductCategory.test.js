import { render } from "@testing-library/react";
import ProductCategory from '../components/ProductCategory';
import { MemoryRouter } from 'react-router-dom';

describe('test ProductCategory components', () => {
  it('should test ProductCategory component', async() => {
    const {getByText} = render(<MemoryRouter><ProductCategory /></MemoryRouter>);
    expect(getByText('Loading...')).toBeInTheDocument();
  });
});
