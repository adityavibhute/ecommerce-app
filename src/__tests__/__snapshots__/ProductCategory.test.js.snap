// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ProductCategory components should test ProductCategory component 1`] = `
<span
  class="sr-only"
>
  Loading...
</span>
`;
